plugins/reporters/web-app/src/funTest/kotlin/WebAppReporterFunTest.kt

```diff
@@ -31,7 +31,7 @@ import org.ossreviewtoolkit.utils.test.readOrtResult
 class WebAppReporterFunTest : WordSpec({
     "WebAppReporter" should {
         "successfully export to a web application" {
-            val ortResult = readOrtResult(getAssetFile("dummy-expected-output-for-analyzer-result.yml"))
+            val ortResult = readOrtResult(getAssetFile("input-analyzer-result.yml"))
```

Comment:
As this file also contains a scan result, I propose to instead use e.g. "scan-result-for-synthetic-gradle-lib.yml".